genesyscloud/resource_genesyscloud_group.go

```diff
@@ -165,14 +165,19 @@ func createGroup(ctx context.Context, d *schema.ResourceData, meta interface{})
 		return diag.Errorf("%v", err)
 	}
 
+	ownerArray := lists.BuildSdkStringListFromInterfaceArray(d, "owner_ids")
+	if len(*ownerArray) < 1 {
+		return diag.Errorf("At least one user required to create group")
+	}
```

Comment:
Instead of implementing this logic yourself, you can set the field `MinItems: 1` in the schema for owner_ids. Then terraform will take care of it